docs-e2e-testing/cypress.config.ts

```diff
@@ -4,6 +4,7 @@ export default defineConfig({
   e2e: {
     supportFile: "support/e2e.{js,jsx,ts,tsx}",
     specPattern: "e2e/**/*.cy.{js,jsx,ts,tsx}",
+    chromeWebSecurity: false,
```

Comment:
Can you elaborate more (and the reference link would be great)?

Reply:
This is a workaround recommended by Crypess to handle the superdomain switch limitation. But I've just read more about this, and it turns out there are more solutions for this problem. Disabling ChromeWebSecurity is the least recommended.
You can read more about this here
1. [Workaround for External Navigation](https://docs.cypress.io/guides/guides/web-security#External-Navigation)
2. [Disabling-Web-Security](https://docs.cypress.io/guides/guides/web-security#Disabling-Web-Security)